houston/ardu/connection.py

```diff
@@ -65,13 +65,19 @@ class MAVLinkConnection(Connection[MAVLinkGeneralMessage]):
     """
     def __init__(self,
                  url: str,
-                 hooks: HOOK_TYPE = None
+                 hooks: HOOK_TYPE = None,
+                 timeout: int = 30
                  ) -> None:
         super().__init__(hooks)
         self.__conn = dronekit.connect(url,
-                                       wait_ready=True,
+                                       wait_ready=False,
                                        heartbeat_timeout=0)
-        self.__conn.wait_ready('autopilot_version')
+        self.__conn.wait_ready(True,
```

Comment:
How come there's two calls to `wait_ready`?

Reply:
`wait_ready` with `True` as argument will wait on all default ready variables and `autopilot_version` is not one of them. So the second `wait_ready` is to also wait for `autopilot_version` (I don't exactly know why we need to wait for `autopilot_version` but it was already there so I assumed there's a reason).